lib/client-assets.php

```diff
@@ -874,10 +874,11 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 	$allowed_block_types = apply_filters( 'allowed_block_types', true );
 
 	$editor_settings = array(
-		'alignWide'        => $align_wide || ! empty( $gutenberg_theme_support[0]['wide-images'] ), // Backcompat. Use `align-wide` outside of `gutenberg` array.
-		'colors'           => $color_palette,
-		'blockTypes'       => $allowed_block_types,
-		'titlePlaceholder' => apply_filters( 'enter_title_here', __( 'Add title', 'gutenberg' ), $post ),
+		'alignWide'          => $align_wide || ! empty( $gutenberg_theme_support[0]['wide-images'] ), // Backcompat. Use `align-wide` outside of `gutenberg` array.
+		'availableTemplates' => wp_get_theme()->get_page_templates( $post_to_edit['id'] ),
```

Comment:
`get_page_templates()` requires a `WP_Post`, sending it an ID does work, but it doesn't convert the ID to a `WP_Post` internally, for sending to the `theme_{$post_type}_templates` filter.

Reply:
Hi @pento, thank you for the feedback, I added a call to WP_Post::get_instance so we can correctly pass a WP_Post instance to the filters.